recipe/meta.yaml

```diff
@@ -53,19 +53,21 @@ requirements:
     - {{ compiler('c') }}
     - {{ compiler('cxx') }}
     - {{ compiler('cuda') }}
-    - arm-variant * {{ arm_variant_type }}  # [aarch64]
-    - sysroot_{{ target_platform }} 2.17    # [linux]
+    #- arm-variant * {{ arm_variant_type }}  # [aarch64]
+    - sysroot_{{ target_platform }} {{ conda_glibc_ver }}.*    # [linux]
   host:
     - patchelf >=0.12  # [linux]
-    - cuda-version {{ cuda_major }}.0
+    # Note that this is now usable with all cuda 12.x (see run pinning), so doesn't need to be
+    # rebuilt with a cuda update
+    - cuda-version {{ cuda_major }}.4
     - libcublas  # [(cuda_compiler_version or "").startswith("12")]
   run:
     - {{ pin_compatible("cuda-version", max_pin="x") }}
     - cudatoolkit >=11.1,<12  # [(cuda_compiler_version or "").startswith("11")]
     - libcublas               # [(cuda_compiler_version or "").startswith("12")]
   run_constrained:
     # Only GLIBC_2.17 or older symbols present
-    - __glibc >=2.17      # [linux]
+    - __glibc >={{ conda_glibc_ver }}      # [linux]
     - arm-variant * {{ arm_variant_type }}  # [aarch64]
 
 test:
```

Comment:
Should we constrain `git  # [win]` as it's available on our win-64 builders? I'm not sure if cloning a repo will cause the build machine to get stuck. It happened with `git checkout <git commit>`

Reply:
I guess you mean `# [not win]` ?

Reply:
Yeah, sorry for confusion, I meant  `# [not win]`